com.unity.cinemachine/Samples~/Shared Assets/CinemachineInputActions.inputactions

```diff
@@ -2,7 +2,7 @@
     "name": "CinemachineInputActions",
```

Comment:
We need to think a little more carefully about the design of our input action asset.  We should make standard stuff like move, look, fire, jump, sprint.  We shouldn't have things like "mouse move", "middle click" and "right click".  Also, Player1 and Player2 should have the same actions (though mapped differently).

![image](https://user-images.githubusercontent.com/6424658/218257003-ba5f68e3-bbc0-407c-af14-64dcb23ad1be.png)